app/services/user.ts

```diff
@@ -306,6 +307,18 @@ export class UserService extends PersistentStatefulService<IUserServiceState> {
     this.onboardingService.start({ isLogin: true });
   }
 
+  private voidStreamKey() {
+    const settings = this.settingsService.getSettingsFormData('Stream');
+
+    settings.forEach(subCategory => {
+      subCategory.parameters.forEach(parameter => {
+        if (parameter.name === 'key') parameter.value = '';
+      });
+    });
+
+    this.settingsService.setSettings('Stream', settings);
+  }
```

Comment:
There's a function in the settings service called `setSettingValue` that makes this much easier now.